lib/js/components/SelectList/SelectListItem/SelectListItem.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<div v-ripple class="selectListItem" :class="{ '-selected': isSelected, [`-${size}`]: true }">
+		<ds-icon
+			v-if="iconLeft"
+			class="selectListItem__icon"
+			:icon="iconLeft"
+			:size="ICON_SIZES.X_SMALL"
+			:spinning="state === SELECT_LIST_ITEM_STATES.LOADING"
+		/>
+		<span class="selectListItem__text">{{ label }}</span>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/spacings';
+
+.selectListItem {
+	@include label-l-default-regular;
+
+	background-color: $color-neutral-background-ghost;
+	color: $color-neutral-text-heavy;
+	display: flex;
+	padding: $space-xs;
+	pointer-events: none;
+
+	&.-selected {
+		background-color: $color-neutral-background;
+	}
+
+	&.-medium {
+		@include label-xl-default-regular;
+	}
+
+	&:not(.-selected) {
```

Comment:
don't we need to block `pointer-events` while loading?

Reply:
We already handle it in https://github.com/bethinkpl/wnl-platform/blob/78303e7b68fab14a1861f02e3fb3d994f28a3cc5/resources/assets/js/modules/assessments/components/headers/AssessmentHeaderContainer.vue#L112-L116 but we can have another layer of protection, sure, I'll add it.

Reply:
IMO, it's better to protect it at lowest possible level, to avoid handling the same type of interaction in multiple places (to avoid inconsistencies)